docs/detections/visual-event-analyzer.asciidoc

```diff
@@ -0,0 +1,92 @@
+[[visual-event-analyzer]]
+[role="xpack"]
+== Visual event analyzer 
+
+Elastic Security allows any event detected by Elastic Endpoint to be analyzed using a hierarchical timeline. This enables security analysts to drill down into all related hosts, processes, and other events to aid in their investigations. 
+
+[float]
+[[find-events-analyze]]
+=== Find events to analyze
+
+You can only visualize events triggered by hosts configured with the Elastic Endpoint Security Integration. In KQL, this translates to any event with the `agent.type` set to `endpoint and process.entity`. 
+
+To access events that can be visually analyzed:
+
+1. In Elastic Security, select **Hosts** > **Events**. A list of all your host's events appears in the bottom panel of the host's view.
+
+2. Create a KQL query that filters all `endpoint` detected events by entering `agent.type: endpoint and process.entity_id : *` into the KQL search bar, and then select **Update**.
++
+[role="screenshot"]
+image::images/kql-agent-type.png[]
++
+A list of all `endpoint` related events appears.
+
+3. Select the **Analyze Event** icon next to the event you want to analyze. 
++
+[role="screenshot"]
+image::images/analyze-event.png[]
+
+The **Visual analyzer** view appears.
+
+[role="screenshot"]
+image::images/analyze-event-view.png[]
+
+You can also analyze events from <<timeline-ui,Timelines>>
+
+
+[discrete]
+[[visual-analyzer]]
+=== Visual analyzer
+
+Inside the visual analyzer, each cube represents either a node, process, or alert. Click and drag in timeline view to see all process relationships. 
```

Comment:
Inside the visual analyzer, the data is represented as a graphical tree where each cube represent a process and is connected via a parent child relationship. Click and drag in the timeline view to see all process relationships. @kqualters-elastic or @oatkiller too much?

Reply:
@michaelolo24: Probably too much., just trying to be thorough. I'll delete that last sentence for now.